db/migrate/20210507115801_drop_pulp_certificate_settings.rb

```diff
@@ -0,0 +1,6 @@
+class DropPulpCertificateSettings < ActiveRecord::Migration[6.0]
+  def change
+    Setting.where(name: 'pulp_client_key').delete_all
+    Setting.where(name: 'pulp_client_cert').delete_all
```

Comment:
https://guides.rubyonrails.org/active_record_querying.html#subset-conditions suggests this can be unified to
```suggestion
    Setting.where(name: ['pulp_client_cert', 'pulp_client_cert']).delete_all
```